src/controllers/krate/publish.rs

```diff
@@ -246,9 +250,13 @@ pub async fn publish(app: AppState, req: BytesRequest) -> AppResult<Json<GoodCra
                     "crates.io only allows a maximum number of {max_features} \
                     features or dependencies that another feature can enable, \
                     but the \"{key}\" feature of your crate is enabling \
-                    {num_features} features or dependencies. If you have a \
-                    valid use case needing to increase this limit, please send \
-                    us an email to help@crates.io to discuss the details."
+                    {num_features} features or dependencies.\n\
+                    \n\
+                    Take a look at https://blog.rust-lang.org/2023/10/26/broken-badges-and-23k-keywords.html \
+                    to understand why this restriction was introduced.\n\
+                    \n\
+                    If you have a valid use case needing to increase this limit, \
+                    please send us an email to help@crates.io to discuss the details."
```

Comment:
Sorry, I obviously missed this last time:

```suggestion
                    If you have a use case that requires an increase of this limit, \
                    please send us an email at help@crates.io to discuss the details."
```

Dropped "valid" because it felt redundant — presumably nobody who contacts us thinks their use case is invalid in the first place. :slightly_smiling_face: 

Snapshots will need to be updated as well.